akka-http-core/src/main/scala/akka/http/javadsl/settings/SettingsCompanion.scala

```diff
@@ -10,7 +10,7 @@ trait SettingsCompanion[T] {
    *
    * Java API
    */
-  final def create(system: ActorSystem): T = create(system.settings.config)
+  def create(system: ActorSystem): T = create(system.settings.config)
```

Comment:
@ktoso - do you want to add the `/** INTERNAL API */` comment to the method? At least it will convey the intent that it should only be used/subclassed by us.